### PR TITLE
Adding compatibility for python 3

### PR DESCRIPTION
**Pull request by cakepietoast**
Tested on 2.7.9 and 3.4.2

### CODE REVIEW

File: smart_selects/views.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
 from django.db.models import get_model
 from django.http import HttpResponse
-try:
-    import json
-except ImportError:
-    from django.utils import simplejson as json
+from django.utils.encoding import force_text
+import json
```
Comment (joelcrocker) [inline smart_selects/views.py]: Django 1.4 still supports Python 2.5, which lacks the `json` module. Is there a reason you needed to break compatibility here?

Comment (cakepietoast) [inline smart_selects/views.py]: No reason other than oversight on my part. Added the try/except clause back in.

**Comment by digi604** [comment]
I need a new maintainer... interested in taking over?

**Comment by jimmykobe1171** [comment]
the compatibility problems for python3 have been fixed.

**Pull request closed without merging**